.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
This allows us to open the project in vscode, using a dev container.  Meaning we can install any version of Node inside the container, and not affect our local machines.